WordPress/Jetpack/Resources/release_notes.txt

```diff
@@ -1,5 +1,5 @@
-Good news, everyone—you can now migrate your WordPress site content to Jetpack without a hitch. If you’re new to the app, you’ll also see a smooth opt-in process to help with the transition.
+* [***] [internal] A significant refactor to the app’s architecture was made to allow for the new simplified UI. Regression testing on the app’s main flows is needed. [#19817]
+* [**] [internal] Disable Story posts when Jetpack features are removed [#19823]
+* [*] [internal] Editor: Only register core blocks when `onlyCoreBlocks` capability is enabled [https://github.com/wordpress-mobile/gutenberg-mobile/pull/5293]
+* [**] [internal] Disable StockPhoto and Tenor media sources when Jetpack features are removed [#19826]
```

Comment:
There are only `[internal]` notes. I'll ping the devs internally to double check.